packages/nimble-components/CONTRIBUTING.md

```diff
@@ -294,6 +294,16 @@ const fancyCheckbox = FoundationCheckbox.compose<CheckboxOptions>({
 
 The project uses a code generation build script to create a Nimble component for each icon provided by nimble tokens. The script is run as part of the `npm run build` command, and can be run individually by invoking `npm run generate-icons`. The generated icon components are not checked into source control, so the icons must be generated before running the TypeScript compilation. The code generation source can be found at `nimble-components/build/generate-icons`.
 
+### Export component tag
+
+Every component should export its custom element tag (e.g. `nimble-button`) in a constant like this:
+
+```ts
+export const buttonTag = DesignSystem.tagFor(Button);
+```
+
+Client code can use this to refer to the component in an HTML template and trust that the reference will continue to work if the component name ever changes.
```

Comment:
> trust that the reference will continue to work if the component name ever changes

Not sure I'd make that guarantee. If the element name changes I'm betting this export name would be updated to match, so the reference will explicitly break during compile (which is a good thing!).

```suggestion
Client code can use this to refer to the component in an HTML template and having a dependency on the export will let a compiled application detect if a tag name changes.
```